packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -745,6 +766,13 @@ describe("wrangler secret", () => {
 
 			"
 		`);
+			expect(std.warn).toMatchInlineSnapshot(`
+			"[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1m\`wrangler secret:bulk\` is deprecated and will be removed in a future major version.[0m
+
+			  Please use \`wrangler secret bulk\` instead, which accepts exactly the same arguments.
```

Comment:
Not for this PR of course, but I'm wondering if it's worth using an abstraction to remove the boilerplate from these snapshots. I suppose the pro is that it's better to be explicit about the output, and the cons are that it's less readable and potentially less maintainable.

Reply:
In this case, this was intentional to test the deprecation message shows. But as mentioned on chat, I'm going to make that a bit more explicit like in https://github.com/cloudflare/workers-sdk/pull/5989